monai/networks/utils.py

```diff
@@ -560,6 +563,7 @@ def convert_to_torchscript(
     device: torch.device | None = None,
     rtol: float = 1e-4,
     atol: float = 0.0,
+    is_trace: bool = False,
```

Comment:
Maybe "use_trace" is more clear a name?

Thanks.

Reply:
Ok, will update this later.